src/main/java/com/ibm/cloudant/kafka/connect/ListRecommender.java

```diff
@@ -37,7 +36,6 @@ public ListRecommender(Object... validValues) {
                 this.validValues.add(validValue);
             }
         }
-        this.validValues = Arrays.asList(validValues);
```

Comment:
this was a bug lurking from the previous PR which meant certain combinations of case-insensitivity didn't work